flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/serde/JsonSerdeUtil.java

```diff
@@ -21,8 +21,7 @@
 import org.apache.flink.table.catalog.ObjectIdentifier;
 import org.apache.flink.table.connector.ChangelogMode;
 import org.apache.flink.table.planner.plan.logical.LogicalWindow;
-import org.apache.flink.table.planner.plan.nodes.exec.ExecNode;
-import org.apache.flink.table.planner.plan.utils.ReflectionsUtil;
```

Comment:
Given you removed this, can you check if you can remove the reflections dependency once for all?

Reply:
It's still used in `MetadataHandlerConsistencyTest` and `JsonSerdeCoverageTest` so I guess I can move it to a test package?

Reply:
yep it's already a beginning. When removing, check if there is any exclusion in `PlannerModule` in planner-loader

Reply:
Done!